lib/csv2avro.rb

```diff
@@ -1,9 +1,19 @@
+require 'logger'
 require 'csv2avro/converter'
 require 'csv2avro/version'
+require 'log/event'
+require 'log/json_formatter'
 
 class CSV2Avro
   attr_reader :input_path, :schema_path, :bad_rows_path, :stdout_option, :options
 
+  def self.log
+    @log ||= Logger.new(STDOUT).tap do |log|
+      log.formatter = Log::JSONFormatter.new
+      log.progname = 'csv2avro'
```

Comment:
`log.formatter = Log::JSONFormatter.new('csv2avro')` is better as it requires passing the application name.